tests/client/ui/components/TestOverlay.js

```diff
@@ -39,6 +39,18 @@ describe('<Overlay />', () => {
     assert.include(root.overlayContainer.className, 'Overlay--visible');
   });
 
+  it('hides when you click the background', () => {
+    const root = render({ visibleOnLoad: true });
+    Simulate.click(root.overlayBackground);
+    assert.notInclude(root.overlayContainer.className, 'Overlay--visible');
+  });
+
+  it('hides when you touch the background', () => {
+    const root = render({ visibleOnLoad: true });
+    Simulate.touchEnd(root.overlayBackground);
```

Comment:
Sometimes I forget with things like `touchEnd`, but if a touch starts outside the background but finishes on it, will it still hide the background? Because that seems wrong.

I'd have thought `onClick` was enough here, but maybe I'm wrong.